tests/template/config.yaml

```
@@ -34,6 +34,7 @@ buffer:
       path: 'placeholder'
       split: 'train'
     default_workflow_type: ''
+    default_eval_type: ''
```

```suggestion
    default_eval_workflow_type: ''
```
We suggest using `default_eval_workflow_type` to keep consistency within the framework.